### PR TITLE
add application variables for lae

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

**Comment by kayiwa** [comment]
this has been run on staging and production

File: group_vars/lae/production.yml [hunk 1]
```diff
@@ -23,48 +29,48 @@ rails_app_dependencies:
   - autoconf
 lae_db_name: 'lae_production'
 lae_db_user: 'lae'
-lae_db_password: '{{vault_lae_db_password}}'
-lae_rabbit_user: '{{vault_lae_rabbit_user}}'
+lae_db_password: '{{ vault_lae_db_password }}'
+lae_rabbit_user: '{{ vault_lae_rabbit_user }}'
```
Comment (hackartisan) [inline group_vars/lae/production.yml:33]: rabbit user and password for prod and staging need to be updated to use the new user and password. see e.g. https://github.com/pulibrary/princeton_ansible/pull/3242/files

It's always been one password, but each app was storing it separately in its own group vars file. with the password rotation it was made more explicit that it's a single password and all the group vars should reference the central password.

Comment (hackartisan) [reply group_vars/lae/production.yml:33]: I have a PR ready to do this.

**Comment by hackartisan** [comment]
ran this latest commit, as well.

**Review by hackartisan — APPROVED** [review]

**Pull request merged**